inspirehep/modules/references/processors.py

```diff
@@ -156,7 +156,6 @@ def add_raw_reference(self, raw_reference, source='reference_builder',
         self._ensure_field('raw_refs', [])
         self.obj['raw_refs'].append({
             'value': raw_reference,
-            'source': source,
```

Comment:
no, those are two different things. `source` is where the raw reference came from, `schema` in what format it is encoded.